README.md

```diff
@@ -1,44 +1,108 @@
+[scratch-gui](https://github.com/LLK/scratch-gui). This is because scratch-gui can be run on its
+own, without needing any other services to be running, while scratch-www needs to communicate
+with several backend systems that the Scratch team runs (see "How this fits in with other Scratch
+repos" above). If you are new to contributing to Scratch's source code, we suggest you start
+by becoming familiar with scratch-gui and its [list of open issues labeled "help
+wanted"](https://github.com/LLK/scratch-gui/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22).
+
+To contribute, please follow the [standard steps for contributing to a project on
+GitHub](https://github.com/firstcontributions/first-contributions).
+
+### Using scratch-www for your own project
+
+If you're thinking of using scratch-www for your own project, you might want to think about what parts
+of the codebase you really need for your project. Is it the layout of the project page? Is it
+the way the project page manages the information it passes to the scratch-gui instance? You may wish
+to use just those parts, and work out the rest your own way.
+
+### License
+
+See the [LICENSE](https://github.com/LLK/scratch-www/blob/master/LICENSE) file in this repo.
+
+## Understanding this codebase
+
+### Guides
+
+Here are some resources to help you get acquainted with how we’re working on the Scratch codebase:
 
 * [Style Guide](https://github.com/LLK/scratch-www/wiki/Style-Guide)
 * [Testing Guide](https://github.com/LLK/scratch-www/wiki/Testing-Guide-for-Bugfixes)
 * [Localization Guide](https://github.com/LLK/scratch-www/wiki/Localization-Guide)
 * [Map of the repository](https://github.com/LLK/scratch-www/wiki/Repo-Map)
 
+### Core technologies
+
+Significant core technologies this codebase uses include:
+
+#### Development technologies
+
+* [Node](https://nodejs.org/)
+* [Webpack](https://webpack.js.org/)
+* [React](https://facebook.github.io/react/)
+* [Redux](https://redux.js.org/)
+* [Sass](http://sass-lang.com/documentation/file.SASS_REFERENCE.html)
+
+#### Testing technologies
+
+Our tests use:
+
+* [Jest](https://jestjs.io/) (we are writing most new tests in Jest)
+* [Tap](https://node-tap.org/) (we are moving away from using Tap, but many tests still use it)
+* [Enzyme](https://airbnb.io/enzyme/)
+* [Selenium](https://selenium.dev/)
+
+## Developing scratch-www
 
 ### Before Getting Started
-* Make sure you have node (v4.2 or higher) and npm [installed](https://docs.npmjs.com/getting-started/installing-node)
-We use npm (Node Package Manager) to maintain and update packages required to build the site.
 
-### Update Packages
-It's important to make sure that all of the dependencies are up to date because the scratch-www code only works with specific versions of the dependencies. You can update the packages by running this command:
+Make sure you have installed:
+* [node](https://docs.npmjs.com/getting-started/installing-node): v4.2 or higher
```

Comment:
Node.js versions older than 8 are EOL and no longer supported.
`npm install` fails outright on Node 4:
![image](https://user-images.githubusercontent.com/25993062/69931305-c02b4d00-1494-11ea-8324-7ce3f2595a68.png)

Maybe bump this to:
```markdown
* [node](https://docs.npmjs.com/getting-started/installing-node): v8 or higher
```

Reply:
Thanks, changed